src/components/EventDetail/index.tsx

```diff
@@ -0,0 +1,102 @@
+import styled from "@emotion/styled";
+import Image from "next/image";
+import Comment from "../Comment";
+import ReplyComment from "../Comment/ReplyComment";
+import Product from "../Product";
+
+const EventDetail = () => {
+  return (
+    <EventContainer>
+      <Event>
+        <li className="event-title">이벤트 제목</li>
+        <li className="event-period">
+          2020-03-01 ~ 2022-04-01
+          <Image
+            src="/assets/share.png"
+            alt="공유 이미지"
+            width="40px"
+            height="40px"
+          />
+        </li>
+        <Image
+          src="/assets/sample_event_detail.png"
+          alt="이벤트 상세 사진"
+          width={958}
+          height={400}
+        />
+        <li className="event-content">
+          Lorem ipsum dolor sit amet consectetur adipisicing elit. In
+          perspiciatis atque voluptate. Veritatis inventore voluptatibus, dicta
+          perspiciatis, magni tenetur, cum ipsa facilis illum impedit adipisci
+          accusantium iure. Temporibus, tempora natus.
+        </li>
+      </Event>
+      <h3>관련 상품</h3>
+      <RelativeProducts>
+        <Product />
+        <Product />
+        <Product />
+      </RelativeProducts>
+      <Comments>
+        <h3>댓글</h3>
+        <hr />
+        <div>
+          <ReplyComment />
+        </div>
+      </Comments>
+    </EventContainer>
+  );
+};
+
+const EventContainer = styled.section`
+  padding: 80px;
+  h3 {
+    font-size: 2.4rem;
+    font-weight: 700;
+    line-height: 34px;
+    margin-bottom: 34px;
+  }
+`;
```

Comment:
p3: 저번에 말씀해주신 css property 통일성이 이 컴포넌트에는 없는거 같습니다. 아래 .event-content에는 margin이 위에 있는데 여기는 margin이 font-size밑에 있네요! 어떤 식으로든 통일성만 지켜지면 좋지 않을까 생각합니다.
```suggestion
  h3 {
   margin-bottom: 34px;
    font-size: 2.4rem;
    font-weight: 700;
    line-height: 34px;
  }
`;
```

Reply:
반영했습니다~~놓쳤네요 ㅠㅠ